functions-methods.js

```diff
@@ -33,4 +54,24 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email) {
+
+    if(email.includes("@") && email.includes(".")) {
+        if (email.indexOf("@") < email.indexOf(".")) {
+            return true
+
+        } else {
+            return false
+        }
+    } else {
+        return false
+    }
```

Comment:
Je zou dit wat kunnen versimpelen waardoor het tegelijk ook wat duidelijker wordt
```suggestion
    return email.includes("@") && email.includes(".") && email.indexOf("@") < email.indexOf(".");
```